lib/ModelMixins/DiscretelyTimeVaryingMixin.ts

```diff
@@ -38,7 +38,7 @@ function DiscretelyTimeVaryingMixin<
     @computed
     get currentTime(): string | undefined {
       const time = super.currentTime;
-      if (time === undefined) {
+      if (time === undefined || time === null) {
```

Comment:
this is just a suggestion, but would it be possible to do something like
```
export const MISSING = undefined;
export const UNSET = null;
```
in Model.ts or some other more 'global' file?
that way this line can read as
`if(time === Model.MISSING || time === Model.UNSET)`

Reply:
I could do that but I think because `undefined` and `null` or so ubiquitous no one might use it? But it gives me another idea, I can maybe define a method called `unsetTrait` which does the right thing by setting the value to `null`.